src/js/lib/client/src/index.js

```diff
@@ -267,6 +273,30 @@ Electron.app.on('web-contents-created', (event,contents) => {
 
 
 
+// =======================
+// === Project Manager ===
+// =======================
+
+const child_process = require('child_process')
+const fss = require('fs')
```

Comment:
move them to top of the file, or refactor this to separate file. Otherwise, its too easy in JS to not notice that something was already imported in this file.